docs/en/operation/Basic-Logging.md

```diff
@@ -51,7 +76,7 @@ For example, if you would like to modify the level for all logs, then you can ch
 log4j.rootLogger=INFO, ${alluxio.logger.type}, ${alluxio.remote.logger.type}
 ```
 
-If you wish to have `DEBUG` logging, then you would make the first line
+To have `DEBUG` logging, modify the first line
```

Comment:
`modify the line to be`
not sure why it's referred to as the first line when the previous statement does not refer to it in the same way

Reply:
refactored